.cache/v/cache/lastfailed

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
can you purge these files completely from the history - they should not be added to the repository